.pipelines/restore-acquiring_code.cmd

```diff
@@ -0,0 +1,89 @@
+@echo off
+
+REM Prerequisities
+echo.
+echo Deleteing python from path...
+set PYTHONPATH=
+set PYTHONHOME=
+
+echo.
+echo Installing Visual Stuio 2019 workloads...
+c:
+mkdir c:\Downloads
+cd c:\Downloads
+curl -o vs_enterprise.exe -L "https://aka.ms/vs/16/release/vs_enterprise.exe"
+call C:\Downloads\vs_enterprise.exe modify --installPath "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise" --add Microsoft.VisualStudio.Component.VC.ATLMFC;includeRecommended --quiet --wait
+
+echo.
+echo Installing Windows Debugging Tools...
+curl -o winsdksetup.exe -L "https://go.microsoft.com/fwlink/p/?linkid=2083338&clcid=0x409"
+winsdksetup.exe /features OptionId.WindowsDesktopDebuggers /quiet
```

Comment:
What happens if machine have more than one SDK installed? The UWP workload from VS installer might install a SDK other that 18362 in near future.

Reply:
This installs the Debugging Tools alone, without the SDK.

Reply:
For which SDK version?

Reply:
Added comment in the previous line that this installs the Debugging Tools alone for Windows 10 SDK, version 1903.